### PR TITLE
feat(perf-issues): Add disabled reason to group actions

### PR DESCRIPTION
**Pull request by evanpurkhiser**

### CODE REVIEW

File: static/app/views/organizationGroupDetails/actions/index.tsx [hunk 1]
```diff
@@ -475,17 +469,20 @@ class Actions extends Component<Props, State> {
...
                   key: 'delete-and-discard',
                   priority: 'danger',
                   label: t('Delete and discard future events'),
                   hidden: !hasAccess,
-                  disabled: !getIssueCapability(group.issueCategory, 'deleteAndDiscard')
-                    .enabled,
-                  onAction: () => this.openDiscardModal(),
+                  disabled: !deleteDiscardCap.enabled,
+                  tooltip: deleteDiscardCap.disabledReason,
+                  onAction: deleteDiscardCap.enabled
```
Comment (evanpurkhiser) [inline static/app/views/organizationGroupDetails/actions/index.tsx:483]: Gonna clean this up once @vuluongj20 fixes up the issues with disabled dropdown items still running the action

File: static/app/views/organizationGroupDetails/actions/index.tsx [hunk 2]
```diff
@@ -475,17 +469,20 @@ class Actions extends Component<Props, State> {
                   priority: 'danger',
                   label: t('Delete'),
                   hidden: !hasAccess,
-                  disabled: !getIssueCapability(group.issueCategory, 'delete').enabled,
-                  onAction: () => this.openDeleteModal(),
+                  disabled: !deleteCap.enabled,
+                  tooltip: deleteCap.disabledReason,
+                  onAction: deleteCap.enabled ? () => this.openDeleteModal() : undefined,
```
Comment (0Calories) [inline static/app/views/organizationGroupDetails/actions/index.tsx:474]: massive nit: we should use `null` for things we're setting ourselves 😄

**Review by 0Calories — APPROVED** [review]

File: static/app/views/organizationGroupDetails/actions/index.tsx (hunk 2 above)
Comment (0Calories) [reply static/app/views/organizationGroupDetails/actions/index.tsx:474]: won't matter once `onAction` is fixed to take `disabled` into account tho

Comment (evanpurkhiser) [reply static/app/views/organizationGroupDetails/actions/index.tsx:474]: I don't think the types will allow for it, and I don't think they should allow for it :)

Setting this as undefined is effectively leaving it as unset.

Comment (0Calories) [reply static/app/views/organizationGroupDetails/actions/index.tsx:474]: That's a good point, agreed!

**Pull request merged**